changelogs/current.yaml

```diff
@@ -214,3 +214,6 @@ deprecated:
   change: |
     deprecated runtime key ``overload.global_downstream_max_connections`` in favor of :ref:`downstream connections monitor
     <envoy_v3_api_msg_extensions.resource_monitors.downstream_connections.v3.DownstreamConnectionsConfig>`.
+- area: upstream
```

Comment:
Please move to hte `removed_config_or_runtime` section above.

Reply:
ah right, good spot.